pkg/urbit/vere/lord.c

```diff
@@ -366,17 +366,15 @@ _lord_plea_slog(u3_lord* god_u, u3_noun dat)
 static void
 _lord_plea_flog(u3_lord* god_u, u3_noun dat)
 {
-  u3_noun pri, tan;
-  c3_w pri_w;
-
   if ( c3n == u3a_is_atom(dat) ) {
-    return _lord_plea_foul(god_u, c3__slog, dat);
+    return _lord_plea_foul(god_u, c3__flog, dat);
   }
 
-  c3_c* tan_c = u3r_string(tan);
+  c3_c* tan_c = u3r_string(dat);
   u3C.stderr_log_f(tan_c);
   c3_free(tan_c);
-  u3z(dat);
+
+  god_u->cb_u.slog_f(god_u->cb_u.ptr_v, 0, dat);
```

Comment:
I don't think we want to print the `%flog` messages twice, isn't that what happens here?

Reply:
Oh, you're right, I got my callbacks confused. Looks like this indeed double-prints in the real terminal. Do we care about explicitly doing `stderr_log_f` here, or are we fine with saying "let the slog callback handle this"?

Reply:
It adds an extra `\n`, which isn't very nice... Probably best to just call the webslog callback directly here.

Reply:
This looks good, we can tweak it later.